README.md

```diff
@@ -1,6 +1,79 @@
 # Eclipse Che on OpenShift
 
-## How to build openshift-connector branch
+## How to build the RedHat Che distribution
+
+#### What is the RedHat Che distribution
+
+The RedHat distribution of Eclipse Che is a RedHat-specific packaging of the Che assemblies
+that allows adding some RedHat specific plugins / behaviors upton the standard upstream Che
+distribution. It is currently based on the `openshift-connector` branch of the upstream
+Che repository.
+
+RedHat modifications against the upstream Che include:
+- the ability to disable the Dashboard (and remove the *Go to Dashboard* button from the Che IDE)
+- The Keycloak integration
+- The Bayesian integration (soon)
+
+#### How the RedHat Che distribution build works
+
+The default behavior is to:
+- checkout the upstream GitHub `che-dependencies` and `che` repositories into the
+`target/export`, based on a given fork (`eclipse` by default) and branch
+(`openshift-connector` by default)
+- build the upstream repositories first
```

Comment:
> For me, it doesn't make sense. It's like we don't have maven ... checkout and build the other repositories should be an option not the default behaviour. The default behaviour should be to rely on maven to get the dependencies.

well, it seems I wasn't clear enough in the `Readme`: what it means really is that *by default*, maven *automatically takes care of everything as part of the root maven build*: it checks out the upstream che, builds it as part of the invoker *setup* step, and then runs the RH assembly sub-build.
I'll update the `Readme` to clarify this point.